README-FR.md

```diff
@@ -1,6 +1,8 @@
 # CYLang
 
-Le langage de programmation basé sur le pseudo-code de CYTech
+Le langage de programmation basé sur le pseudo-code de CYTech.   
+
+Read the English version [here.]()
```

Comment:
The link to the english README is missing. Can you correct it ? Once done I'll merge the PR.